### PR TITLE
Feature/kids 1223 add login to recommendation flow

### PR DESCRIPTION
**Pull request by pavelpavelko**
## Description
- Start screen is redesigned
- Login screen is redesigned
- Several elevated buttons implemented (for future reuse)
- Login feature is integrated into the recommendation flow
- Amplitude updated

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/shared/widgets/givt_tertiary_elevated_button.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+class GivtTertiaryElevatedButton extends StatelessWidget {
+  const GivtTertiaryElevatedButton({
+    required this.text,
+    required this.onPressed,
+    this.isLoading = false,
+    super.key,
+  });
+
+  static const Color _tertiaryBackgroundColor = Color(0xFFE28D4D);
```
Comment (MaikelStuivenberg) [inline lib/shared/widgets/givt_tertiary_elevated_button.dart:12]: Wouldn't it be better to have those colors in one separated file?

File: lib/shared/widgets/givt_secondary_elevated_button.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:givt_app_kids_web/shared/widgets/givt_elevated_button.dart';
+
+class GivtSecondaryElevatedButton extends StatelessWidget {
```
Comment (MaikelStuivenberg) [inline lib/shared/widgets/givt_secondary_elevated_button.dart:4]: Nice how you made components out of the different buttons :)

File: lib/shared/widgets/bubble.dart [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    }
+  }
+}
+
+enum BubbleDirection {
+  above,
+  atRight,
+}
+
+class DrawTriangle extends CustomPainter {
```
Comment (MaikelStuivenberg) [inline lib/shared/widgets/bubble.dart:71]: Should we move this to a seperate file?
Personally i dont like to have multiple (long) classes in one file.

File: lib/shared/widgets/givt_tertiary_elevated_button.dart (hunk 1 above)
Comment (pavelpavelko) [reply lib/shared/widgets/givt_tertiary_elevated_button.dart:12]: yes, initially I thought to create apptheme file (as in givt app) and move them there. 
However I decided to wait until we will have more clarity about the themeing/colors approach we are working on at the moment with Elmar. So the code is prepared for such change soon.

File: lib/shared/widgets/givt_secondary_elevated_button.dart (hunk 2 above)
Comment (pavelpavelko) [reply lib/shared/widgets/givt_secondary_elevated_button.dart:4]: thanks :)

**Pull request merged**